libs/financial-aid/shared/src/components/FileList/FileList.tsx

```diff
@@ -9,6 +8,17 @@ import {
   getFileType,
 } from '@island.is/financial-aid/shared/lib'
 
+import { gql, useLazyQuery } from '@apollo/client'
+
+export const GetSignedUrlQuery = gql`
+  query GetSignedUrlQuery($input: GetSignedUrlForIdInput!) {
+    getSignedUrlForId(input: $input) {
+      url
+      key
+    }
+  }
+`
```

Comment:
Know this should be in sharedGql, but if this is there then we have a circular dependency 😭  We could add a sharedGql file into our shared lib, but this is the only on there so putting it on hold for now 🧑‍🤝‍🧑